File: AUGUST-2025/14-08-2025/p2264.swift
class Solution {
    func largestGoodInteger(_ num: String) -> String {
        var curr: Array<Character> = ["x", "y", "?"]
        var ans = -1

        for c in num {
            curr[0] = curr[1]
            curr[1] = curr[2]
            curr[2] = c
            
            if curr[0] == curr[1] && curr[1] == curr[2] && Int(String(curr[0]))! > ans {
                ans = Int(String(curr[0]))!
            }
        }
        
        if ans != -1 {
            return String(repeating: String(ans), count: 3)
        }
        
        return ""
    }
}

File: AUGUST-2025/14-08-2025/p2264.cpp
class Solution {
public:
    string largestGoodInteger(string num) {
        for (char d = '9'; d >= '0'; d--) {
            string triple(3, d);
            if (num.find(triple) != string::npos) {
                return triple;
            }
        }
        return "";
    }
};